Codeinwp/neve
# Development

### Summary

Exclude Node modules from phpcs
Tags removed from the archive page. Description added next to the title on archive page.
Small changes for compatibility with WooCommerce.
Remove theme support for header image (custom-header).
Style reviews comment-form on product page when user is logged out.

### Will affect visual aspect of the product

YES

### Screenshots 
Archive
<img width="1251" alt="neve-archive" src="https://user-images.githubusercontent.com/26867648/47012015-f18b7100-d14b-11e8-84bf-4995e10c59fe.png">

Cart
<img width="1263" alt="neve-cart" src="https://user-images.githubusercontent.com/26867648/47224700-cbbcd100-d3c4-11e8-8f48-8b8aa29cc5ce.png">

Checkout
<img width="1479" alt="neve-checkout" src="https://user-images.githubusercontent.com/26867648/47012033-f7815200-d14b-11e8-877a-bd662c1a082d.png">

Reviews
![neve-reviews](https://user-images.githubusercontent.com/26867648/47075634-a03fb800-d205-11e8-884d-9ca34089a2e9.png)

### Test instructions


- Check the archive page, Category, Author, Tag
- Check cart page, take a look at cart-actions buttons, and cart totals table/widget.
- Check the coupon button on checkout page, try it on responsive
- Check the product page, reviews comment form when user is logged out.


Closes #46, #92, #60
